src/CMakeLists.txt

```diff
@@ -229,3 +177,22 @@ target_include_directories(parthenon PUBLIC
 )
 
 lint_target(parthenon)
+
+# Generate install targets for make if CMAKE_INSTALL_PREFIX has been provided
+if(NOT CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  install(TARGETS parthenon LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+    PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/parthenon")
+
+  # Maintain directory structure in installed include files
+  install(FILES ${base_hdr} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+  foreach( subdir ${parthenon_subdirs} )
+    file(TO_CMAKE_PATH "${subdir}" hdr_subdir)
+    install(DIRECTORY "${hdr_subdir}" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+    FILES_MATCHING PATTERN "*.hpp")
```

Comment:
It would be nice if we implemented #239 because then you could just copy a specific directory of public headers, but it's not in scope for this change.